cal-itp/benefits
# Eligibility Index: Veterans - Use Older Adult copy

This PR depends on #1705 to be merged first.

## What this PR does
- Copy: On Elig Index, for the Veteran pathway, change the last sentence to read: `You will need to verify your identity with Login.gov.`, with a link to the Login.gov modal. *Both* links from the Veteran and the Older Adult radio input open the _same exact_ modal.
- Templates: Like in #1705, rename senior modal to login-gov modal.

## Includes, CSS changes
- CSS: Convert the `#login` id to a class. This is necessary b/c now, for the first time, a page has more than one Login.gov button on it. This requires changing _all_ calls to the `#login` id to the class. This affects the Eligibility Start page for Veterans, Eligibility Start page for Older Adults, and the Enrollment Success page.
- Modal trigger element: Previously the Modal trigger element needed an `id` field to pass in the `id` of `login`, to show the color Login.gov trigger image. Now, though, there is no `id` necessary to pass in. So I removed the field entirely from the includes template. It was actually causing an interesting bug on the radio button field page, where the id of the _radio button field_ was being used in the modal trigger instead - which was resulting in 2 links with the same id. And that was coming up in a11y testing.

## How to test this PR
- Test the regular app flow
- Pay attention to all buttons with the Login.gov icon: Elig Start, Log out on Enrollment Success, Login.gov modal triggers. Ensure no visual regressions.

<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/b5a93ef2-cee7-4a97-9dcd-bbba93f480a3">
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/7acf7ebd-23ac-463f-b6fd-59ece68b5920">
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/e792d58f-ecc6-4bfa-968b-aa32dede8ee2">
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/3939bd9b-d5fd-409a-9337-5d0e27528777">
